keyboards/minidox/keymaps/norman/readme.md

```diff
@@ -0,0 +1,56 @@
+# Minidox Norman Layout by LeHoff
+Based on the QWERTY layout created by Khitsule.
+![Khitsule](https://imgur.com/ogNeW42)
```

Comment:
The URL here should be a direct link to an image file, if you want the image displayed on the page.

```suggestion
![Khitsule](https://i.imgur.com/ogNeW42.png)
```

If you just want to *link* to the image, delete the `!` at the start of the line.